### ISSUE TITLE
`# eslint-disable-next-line @graphql-eslint/naming-convention` is marked as `spaced-comment` error

### ISSUE THREAD

**Issue by chenrui333** [dotansimha/graphql-eslint#760]
**Describe the bug**

When I am trying to upgrade @graphql-eslint/eslint-plugin from 0.4.4 to 2.4.0 and found some `spaced-comment` error reports. And one of the following is the below one (I think it should not be marked as error?)

```
type Query @rateLimit {
  # eslint-disable-next-line @graphql-eslint/naming-convention
  _empty: String
}
```

**To Reproduce**

**Expected behavior**

**Environment:**

- OS:
- `@graphql-eslint/...`: 2.4.0
- NodeJS: 14.x

**Additional context**

**Comment by B2o5T** [comment]
Hi @chenrui333 what's your `spaced-comment` configuration?
Please run:
```sh
eslint --print-config path-to-graphql-file
```

**Comment by dotansimha** [comment]
@chenrui333 any update on this?

**Comment by B2o5T** [comment]
closed due inactivity
